### PR TITLE
[ADAM-1517] Bump Parquet version in a manner compatible with Spark 2.2.x

### LINKED ISSUE #1517: Move to Spark 2.2, Parquet 1.8.2
Requires shading Avro.

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #1517.

### CODE REVIEW

**Comment by fnothaft** [comment]
Whoopsed and forgot to touch up the changes to the `move_to_spark1.sh` script. Should be good to go now.

**Comment by fnothaft** [comment]
Jenkins, test this please.

**Comment by fnothaft** [comment]
Spark 2.2.x drops Hadoop 2.3, which is why that build failed. Otherwise, this looks good on my end.

**Comment by fnothaft** [comment]
Jenkins, test this please.

**Comment by heuermh** [comment]
> Spark 2.2.x drops Hadoop 2.3, which is why that build failed.

Should we take this opportunity (before the 0.23.0 release) to move 2.6.0 to 2.7.3 in the Jenkins configuration matrix?

Is there any reason to add Spark 2.1.x?  I don't know of any.

File: pom.xml [hunk 1]
```diff
@@ -515,7 +515,13 @@
       <dependency>
         <groupId>org.apache.parquet</groupId>
         <artifactId>parquet-avro</artifactId>
-        <version>${parquet.version}</version>
+        <version>1.8.1</version>
```
Comment (heuermh) [inline pom.xml]: Could you add a comment here about intentional pinning to version 1.8.1?

**Comment by heuermh** [comment]
>  heuermh dismissed their review 2 minutes from now

Sorry for the noise, that didn't do what I thought it would do.

File: pom.xml (hunk 1 above)
Comment (fnothaft) [reply pom.xml]: +1

**Comment by fnothaft** [comment]
Rebased and added the comment.

> Should we take this opportunity (before the 0.23.0 release) to move 2.6.0 to 2.7.3 in the Jenkins configuration matrix? And bump Spark 1.6.1 to Spark 1.6.3?

+1 and done! Instead of moving 2.6.0, I dropped 2.3.0, added 2.7.3, and bumped 2.6.0 to 2.6.2. I believe there are still some commercial Hadoop distros pushing 2.6.x.

> Then is there any reason to add Spark 2.1.x? I don't know of any.

Agreed.

**Comment by fnothaft** [comment]
From a sequencing perspective, let's merge this PR before any other PRs, since I've bumped the jenkins config.

**Comment by fnothaft** [comment]
Looks like an S3 outage...

**Comment by fnothaft** [comment]
Jenkins, test this please.

**Comment by heuermh** [comment]
Thank you, @fnothaft

**Pull request merged**